homeassistant/components/broadlink/switch.py

```diff
@@ -42,31 +35,17 @@
     }
 )
 
-OLD_SWITCH_SCHEMA = vol.Schema(
+PLATFORM_SCHEMA = vol.Schema(
     {
-        vol.Optional(CONF_COMMAND_OFF): data_packet,
-        vol.Optional(CONF_COMMAND_ON): data_packet,
-        vol.Optional(CONF_FRIENDLY_NAME): cv.string,
+        vol.Required(CONF_MAC): mac_address,
+        vol.Optional(CONF_HOST): cv.string,
+        vol.Optional(CONF_SWITCHES, default=[]): vol.All(
+            cv.ensure_list,
+            [SWITCH_SCHEMA],
+        ),
     }
```

Comment:
Look good! One last thing, I made a mistake here. We need to extend the default platform schema:

```python3
PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
    {
        vol.Required(CONF_MAC): mac_address,
        vol.Optional(CONF_HOST): cv.string,
        vol.Optional(CONF_SWITCHES, default=[]): vol.All(
            cv.ensure_list,
            [SWITCH_SCHEMA],
        ),
    }
 )

Reply:
We should leave the `PLATFORM_SCHEMA` as-is since its still in use and the keys are still deprecated.  Once `switch` is using a config flow, we can rip it out at that time.

Reply:
Ok. @tkdrob, uno mas? Hehehe